tests/containers/docker_image.pm

```diff
@@ -23,30 +25,29 @@ use utils;
 use containers::common;
 use containers::container_images;
 use suse_container_urls 'get_suse_container_urls';
-use scheduler 'get_test_suite_data';
+use version_utils qw(get_os_release check_host_os);
 
 sub run {
     my ($image_names, $stable_names) = get_suse_container_urls();
-    my $test_data = get_test_suite_data();
-    my $runtime   = "docker";
+    my ($running_version, $sp, $host_distri) = get_os_release;
+    my $runtime = "docker";
 
-    install_docker_when_needed($test_data->{host_os});
+    install_docker_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
     for my $iname (@{$image_names}) {
         test_container_image(image => $iname, runtime => $runtime);
         build_container_image(image => $iname, runtime => $runtime);
-        unless ($test_data->{host_os}) {
+        if (check_host_os('suse')) {
```

Comment:
not sure if it will work here, but normally we have these `is_suse()` calls, or?

Reply:
i did not find `is_suse` but in general the majority of them (if not all) are relied on the openqa variables. there are jobs like these that we still keep DISTRI with `sle` but the host is something different. So the idea was to identify the host dynamically and independently from the OpenQA variables

Reply:
ok thx for clarification